client/state/data-layer/wpcom-http/test/utils.js

```diff
@@ -18,10 +18,22 @@ describe( 'WPCOM HTTP Data Layer', () => {
 			expect( getData( action ) ).to.equal( data );
 		} );
 
-		it( 'should return null if no response data available', () => {
+		it( 'should return undefined if no response data available', () => {
 			const action = { type: 'SLUGGER' };
 
-			expect( getData( action ) ).to.be.null;
+			expect( getData( action ) ).to.be.undefined;
+		} );
+		it( 'should return an empty string', () => {
+			const data = '';
+			const action = { type: 'SLUGGER', meta: { dataLayer: { data } } };
+
+			expect( getData( action ) ).to.equal( data );
+		} );
+		it( 'should return null', () => {
+			const data = null;
+			const action = { type: 'SLUGGER', meta: { dataLayer: { data } } };
+
+			expect( getData( action ) ).to.equal( data );
```

Comment:
I think that we can make these tests more expressive of their intent:

```js
it( 'should return valid-but-falsey data', () => {
	const withData = data => ( { type: 'SLUGGER', meta: { dataLayer: { data } } } );

	expect( getData( withData( '' ) ).to.equal( '' );
	expect( getData( withData( null ) ).to.equal( null );
	expect( getData( withData( 0 ) ).to.equal( 0 );
	expect( getData( withData( false ) ).to.equal( false );
} );
```

^^^ something like that